.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo python3 -m pip install --upgrade pip
+          pip install -U setuptools
+          sudo pip3 install nbconvert pelican pytest-tornasync
+
+      - name: Install oscovida
+        run: |
+          make dev-install
```

Comment:
Personally I dislike using makefiles for this since it can obfuscate errors and issues if they come up

Reply:
Also, for these tests, the install command should be the same one used by the users, so there's an actual reason for this in addition to my personal dislike of it :stuck_out_tongue:

Reply:
Agree with you. My first thought was to get rig of that, but then I realised that in fact this `Makefile` is more visible to users that a Github action.